camelot/parsers/stream.py

```diff
@@ -95,12 +96,17 @@ def _text_bbox(t_bbox):
             Tuple (x0, y0, x1, y1) in pdf coordinate space.
 
         """
-        xmin = min([t.x0 for direction in t_bbox for t in t_bbox[direction]])
-        ymin = min([t.y0 for direction in t_bbox for t in t_bbox[direction]])
-        xmax = max([t.x1 for direction in t_bbox for t in t_bbox[direction]])
-        ymax = max([t.y1 for direction in t_bbox for t in t_bbox[direction]])
-        text_bbox = (xmin, ymin, xmax, ymax)
-        return text_bbox
+        try:
+            xmin = min([t.x0 for direction in t_bbox for t in t_bbox[direction]])
+            ymin = min([t.y0 for direction in t_bbox for t in t_bbox[direction]])
+            xmax = max([t.x1 for direction in t_bbox for t in t_bbox[direction]])
+            ymax = max([t.y1 for direction in t_bbox for t in t_bbox[direction]])
+            text_bbox = (xmin, ymin, xmax, ymax)
+            return text_bbox
+        except ValueError as err:
```

Comment:
`sys.exit` is not a good way to go. Instead we should catch the exception using `except Exception` and then raised a `ValueError` which says "Table area has no text".